xbmc/addons/GUIViewStateAddonBrowser.cpp

```diff
@@ -40,6 +40,7 @@ CGUIViewStateAddonBrowser::CGUIViewStateAddonBrowser(const CFileItemList& items)
   else
   {
     AddSortMethod(SortByLabel, SortAttributeIgnoreFolders, 551, LABEL_MASKS("%L", "%I", "%L", ""));  // Filename, Size | Foldername, empty
+    AddSortMethod(SortByLastUsed, 552, LABEL_MASKS("%L", "%u", "%L", "%u")); //Label, Last used
```

Comment:
Shouldn't we add last installed and last updated also?
One problem would be letting the user understand that last updated means locally on his system. Not updated in the repo.